contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -164,15 +143,16 @@ void ConfigManager::fetch_configs(std::shared_ptr<ConfigsFetchInfo> fetchInfo) {
                          const std::string& config) {
```

Comment:
how can you pass compiling?  

if config here is const::string&,  it can not be moved.   its type should be string&&